jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1392,6 +1412,43 @@
       (pushback (send *baxter* arm :inverse-kinematics coords-tmp :rotation-axis t) avs)
       (send self :angle-vector-sequence avs :fast nil 0 :scale 4.0)
       ))
+  (:wait-interpolation-until-grasp
+    (arm)
+    (while (send self :interpolatingp)
+           (if (send self :graspingp arm)
+             (progn
+               (ros::ros-info "[:wait-interpolation-until-grasp] Grasping detected. Cancel angle vector: ~a" arm)
+               (send self :cancel-angle-vector)))))
+  (:get-support-arm-coords
```

Comment:
@knorth55 then, why did you decided to place this method in baxter-interface?
I thought this can be baxter-robot class (because this is independent on ROS).